octobot_evaluators/evaluator/strategy_evaluator.py

```diff
@@ -57,6 +128,201 @@ async def matrix_callback(self,
         # Do not forget to check if evaluator_name is self.name
         pass
 
+    def _are_every_evaluation_valid(self,
+                                    matrix_id,
+                                    evaluator_name,
+                                    evaluator_type,
+                                    exchange_name,
+                                    cryptocurrency,
+                                    symbol,
+                                    time_frame
+                                    ):
+        available_time_frames = self.get_available_time_frames(matrix_id, exchange_name, evaluator_type,
+                                                               cryptocurrency, symbol)
+        to_check_time_frames = [tf
+                                for tf in available_time_frames
+                                if TimeFrames(tf) in self.strategy_time_frames]
+        available_evaluators = self._get_available_evaluators(matrix_id, exchange_name, evaluator_type)
+        to_validate_node_paths = [
```

Comment:
Should it be stored in the 'node cache' which we talked about in private ?

Reply:
It could be yes, I will add this